docs/tx_rollup.md

```diff
@@ -0,0 +1,29 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 Oxhead Alpha
+   - SPDX-License-Identifier: LicenseRef-MIT-OA
+   -->
+
+# Setting up transaction rollup node on Ubuntu
+
+`proto` variable can hold every active protocol, currently `013-PtJakart` and `014-PtKathma`.
+
+By default, `tezos-tx-rollup-node-$proto.service` will using:
+* `/var/lib/tezos/.tezos-client` as the `tezos-client` data directory, set in `DATA_DIR`
+* `http://localhost:8732` as the `tezos-node` RPC address, set in `NODE_RPC_ENDPOINT`
+* `127.0.0.1:8472` as the `tezos-tx-rollup-node-$proto` RPC address, set in `ROLLUP_NODE_RPC_ENDPOINT`
+* `observer` as the `tezos-tx-rollup-node-$proto` working mode, set in `ROLLUP_MODE`
+* `rollup` as the rollup alias, set in `ROLLUP_ALIAS`
+
+In order to change defaults, edit `/etc/default/tezos-tx-rollup-node-$proto` file and restart the service.
+
```

Comment:
Perhaps it's worth giving an example of how to do the edit and the restart, see the baking doc for example.